test/serializer_test.exs

```diff
@@ -15,6 +15,12 @@ defmodule JSONAPISerializerTest do
         best_comments: {JSONAPISerializerTest.CommentView, :include}
       ]
     end
+
+    def links(data, conn) do
+      %{
+        next: url_for_pagination(data, conn, %{cursor: "some-string"})
```

Comment:
Also if we're going down the pagination path we MUST support next, prev, first and last.